src/OpenTelemetry.Exporter.ZPages/Implementation/ZPagesSpanInformation.cs

```diff
@@ -0,0 +1,98 @@
+﻿// <copyright file="ZPagesSpanInformation.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using OpenTelemetry.Trace.Export;
+
+namespace OpenTelemetry.Exporter.ZPages.Implementation
+{
+    /// <summary>
+    /// Stores the span information aggregated according to span name.
+    /// </summary>
+    public class ZPagesSpanInformation
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZPagesSpanInformation"/> class.
+        /// </summary>
+        public ZPagesSpanInformation()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZPagesSpanInformation"/> class when span data is provided.
+        /// </summary>
+        /// <param name="spanData">Input span data to be used for initialization.</param>
+        public ZPagesSpanInformation(SpanData spanData)
+        {
+            this.Name = spanData.Name;
+            this.CountTotal = 1;
+            this.EndedCount = 0;
+            this.LastUpdated = new DateTimeOffset(DateTime.Now).ToUnixTimeMilliseconds();
+            this.TotalLatency = 0;
+            this.AvgLatencyTotal = 0;
+            this.ErrorTotal = 0;
+        }
+
+        /// <summary>
+        /// Gets or sets the name of the span.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total count of the span.
+        /// </summary>
+        public long CountTotal { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total count of the ended span.
+        /// </summary>
+        public long EndedCount { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total error count of the span.
+        /// </summary>
+        public long ErrorTotal { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total average latency of the span.
+        /// </summary>
+        public long AvgLatencyTotal { get; set; }
```

Comment:
Why latency is counted as `long`? What the unit of measurement? Perhaps - use `Timespan` type instead of `long`?

Reply:
The unit for latency is milliseconds, hence it was counted as long. The same gets displayed on the HTML. Do we need to use TimeSpan instead?